frontend/src/folder/Modal/CreateFolderModal.tsx

```diff
@@ -1,19 +1,36 @@
 import { FC } from 'react';
 import { ModalProps } from '../types/modal';
-import { AlertDialog, AlertDialogContent } from '@radix-ui/react-alert-dialog';
+import { Dialog, DialogContent, DialogOverlay, DialogClose } from '@radix-ui/react-dialog';
```

Comment:
Not radix dialog directly, but shadcn UI dialog https://ui.shadcn.com/docs/components/dialog

You're importing the radix dialog and styling it yourself

While if you use shadcn dialog — it does all the styling for you